test/integration/imagechange_buildtrigger_test.go

```diff
@@ -528,5 +544,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 		if updatedConfig.Spec.Triggers[tc.triggerIndex].ImageChange.LastTriggeredImageID != "registry:5000/openshift/"+tc.name+":"+tc.tag {
 			t.Fatalf("Expected imageID equal to pull spec, got %#v", updatedConfig.Spec.Triggers[0].ImageChange)
 		}
+
+		ignoreBuilds[newBuild.Name] = ""
```

Comment:
we can't do that.  We can't mark it as seen until the entire test is complete, otherwise we'd ignore the additional events we *are* expecting to see.  It's only safe to mark it as seen at the very end of the loop, so that if someone comes back in and adds more logic in this test case, they won't end up having events get skipped.

I will change it to a map of structs, thanks, not familiar w/ the idiomatic patterns w/ respect to maps.  (also not really sure why it matters, frankly, but that's true of most of the go idioms imho).